mevislab.github.io/content/examples/data_objects/contours/example1/index.md

```diff
@@ -4,11 +4,16 @@ title: "Creation of Contours"
 category: "data_objects"
 ---
 
-# Contour Example 1: Creation of Contours
-This example shows how to create CSOs using SoCSOEditor modules.
+# Contour Example 1: Creation of contours
+Contours are stored as CSOs or Contour Segmented Objects in MeVisLab.
```

Comment:
better: "Contour Segmented Objects (CSOs)", otherwise it may read like different objects

Reply:
Will change that as requested